platform/resources/ValidateSettings.lua

```diff
@@ -52,7 +52,8 @@ function validateSettings(arg1, arg2)
 
 		print = origPrint
 
-		if ok then
+		-- allow developers to optionally use more complex settings files without raising warnings
+		if ok or validSettings.complexFile then
```

Comment:
`validSettings.complexFile` would always be true though, wouldn't it?

Reply:
You are absolutely right. Seems I had run my final tests in a debug build of the simulator, so I didn't catch the issue.